framework/test/jest/integration/specs/modules/chain/loader/validate_own_chain/valid_chain_partial_round.spec.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+describe('validateOwnChain', () => {
```

Comment:
Same comment as for framework/test/jest/integration/specs/modules/chain/loader/validate_own_chain/first_round.spec.js